src/bin/tikv-server.rs

```diff
@@ -198,6 +198,14 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     };
     opts.set_min_write_buffer_number_to_merge(min_write_buffer_number_to_merge as i32);
 
+    let max_background_threads = get_integer_value("",
```

Comment:
My mistake, rocksdb will adjust the maximum number of background threads according to the `max_background_compactions` and `max_background_flushes` options.